install.sh

```diff
@@ -270,28 +263,33 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   rest="${tarfile#*-}"    # extract string after first '-'
   version="${rest%%-chromeos*}"
                           # extract string between first '-' and "-chromeos"
-
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 echo -e "${GRAY}"
+# Work around https://github.com/chromebrew/chromebrew/issues/3305
+sudo ldconfig &> /dev/null || true
+echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
-echo -e "${RESET}"
 
-echo_info "Setup and synchronize local package repo..."
-echo -e "${GRAY}"
+echo_out "Set up and synchronize local package repo..."
 
 # Remove old git config directories if they exist.
 find "${CREW_LIB_PATH}" -mindepth 1 -delete
 
 curl -L --progress-bar https://github.com/"${OWNER}"/"${REPO}"/tarball/"${BRANCH}" | tar -xz --strip-components=1 -C "${CREW_LIB_PATH}"
 
 # Set LD_LIBRARY_PATH so crew doesn't break on i686 and
-# the mandb install doesn't fail.
-export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}"
+# the mandb postinstall doesn't fail.
+echo "export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX" >> $CREW_PREFIX/etc/env.d/path
+export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX:$LD_LIBRARY_PATH
```

Comment:
`export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX`?

Reply:
That would reset LD_LIBRARY_PATH to only be CREW_PREFIX/lib$LIB_SUFFIX

Reply:
Exactly.

Reply:
Wouldn't we want the pre-existing LD_LIBRARY_PATH? I don't think chromebrew should be wiping pre-existing system variables.

Reply:
Yes, because there is no reason it should be anything else; especially during installation.